src/index.html

```diff
@@ -4,9 +4,271 @@
+        </a>
+        <h2 class="container__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+        <div class="container__number">Код товара: 195434</div>
+        <div class="container__stars">
+          <div class="container__stars container_4">
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+          </div>
+          <div class="container__feedback">Отзывов: 5</div>
+        </div>
+        <div class="container__price">
+          <div class="container__price-name">Цена:</div>
+          <div class="container__price-number">69 999 грн</div>
+        </div>
+        <a href="#"
+        class="buy"
+        data-qa="hover"
+        >
+        КУПИТЬ
+        </a>
+      </div>
+      <div class="container">
+        <a href="#">
+          <img src="./images/imac.jpeg"
+            class="container__product"
+            alt="comp">
+        </a>
+        <h2 class="container__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+        <div class="container__number">Код товара: 195434</div>
+        <div class="container__stars">
+          <div class="container__stars container_4">
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+          </div>
+          <div class="container__feedback">Отзывов: 5</div>
+        </div>
+        <div class="container__price">
+          <div class="container__price-name">Цена:</div>
+          <div class="container__price-number">69 999 грн</div>
+        </div>
+        <a href="#"
+        class="buy"
+        >
+        КУПИТЬ
+        </a>
+      </div>
+      <div class="container">
+        <a href="#">
+          <img src="./images/imac.jpeg"
+            class="container__product"
+            alt="comp">
+        </a>
+        <h2 class="container__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+        <div class="container__number">Код товара: 195434</div>
+        <div class="container__stars">
+          <div class="container__stars container_4">
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+          </div>
+          <div class="container__feedback">Отзывов: 5</div>
+        </div>
+        <div class="container__price">
+          <div class="container__price-name">Цена:</div>
+          <div class="container__price-number">69 999 грн</div>
+        </div>
+        <a href="#"
+        class="buy"
+        >
+        КУПИТЬ
+        </a>
```

Comment:
```suggestion
        <a 
          href="#"
          class="buy"
        >
          Купить
        </a>
```
Делай transform в css